test/tools/Makefile

```
@@ -6,7 +6,13 @@
 ARCH                    ?= $(subst x86_64,amd64,$(shell uname -m))
 BUILTIN_CLOUD_PROVIDERS ?= aws azure ibmcloud vsphere libvirt
 GOFLAGS                 ?=
-GOOPTIONS               ?= GOOS=linux GOARCH=$(ARCH) CGO_ENABLED=0
+GOOS                    ?= $(shell uname)
```

```suggestion
GOOS                    ?= $(shell uname | tr A-Z a-z)
```

instead of the if/else below?